@ecomplus/storefront-components/src/html/TheProduct.html

```diff
@@ -253,6 +250,14 @@
               </p>
             </template>
 
+            <!-- Predefined slot for promotion timers -->
+            <slot name="sale-timer">
+              <div v-if="isOnSale" class="sale-timer">
+                <span>Oferta<br>acaba em:</span>
+                <h1 id="promo-time">00:00:00</h1>
```

Comment:
```suggestion
                <div
                  class="h1">
                  id="product-promo-time"
                >
                  00:00:00
                </div>
```

A gente não pode usar `<h1>` aqui porque seria uma cagada pra [SEO](https://resultadosdigitais.com.br/o-que-e-seo/) , h1 sempre tem que ser o título mais relevante para indexação da página, na página de produto o `<h1>` tem que ser exclusivamente o nome do produto.

O id até dá pra usar mas precisa do prefixo `product-` como um escopo pra evitar bater com id de algum outro elemento da página (em outro componente), mas acho que o ideal (faster) seria usar [ref](https://vuejs.org/guide/essentials/template-refs.html) em vez de id aqui, não?

Reply:
ok